qcodes/instrument_drivers/oxford/ILM200.py

```diff
@@ -0,0 +1,328 @@
+        """
+        try:
+            idstr = ''  # in case self.ask fails
+            idstr = self._get_version().split()
+            # form is supposed to be comma-separated, but we've seen
+            # other separators occasionally
+            idparts = [idstr[3] + ' ' + idstr[4], idstr[0], idstr[5],
+                       idstr[1] + ' ' + idstr[2]]
+            # in case parts at the end are missing, fill in None
+            if len(idparts) < 4:
+                idparts += [None] * (4 - len(idparts))
+        except:
+            logging.warn('Error getting or interpreting *IDN?: ' + repr(idstr))
+            idparts = [None, None, None, None]
+
+        return dict(zip(('vendor', 'model', 'serial', 'firmware'), idparts))
+
+    def get_all(self):
+        """
+        Reads all implemented parameters from the instrument,
+        and updates the wrapper.
+
+        Input:
+            None
+
+        Output:
+            None
+        """
+        logging.info(__name__ + ' : reading all settings from instrument')
+        self.level.get()
+        self.status.get()
+        self.rate.get()
+
+    def close(self):
+        """
+        Safely close connection
+        """
+        logging.info(__name__ + ' : Closing ILM200 connection')
+        self.local()
+        super().close()
+
+    # Functions: Monitor commands
+    def _get_version(self):
+        """
+        Identify the device
+
+        Input:
+            None
+
+        Output:
+            'ILM200 Version 1.08 (c) OXFORD 1994\r'
+        """
+        logging.info(__name__ + ' : Identify the device')
+        return self._execute('V')
+
+    def _do_get_level(self):
+        """
+        Get Helium level of channel 1.
+        Input:
+            None
+
+        Output:
+            result (float) : Helium level
+        """
+        logging.info(__name__ + ' : Read level of channel 1')
+        result = self._execute('R1')
+        return float(result.replace("R", "")) / 10
+
+    def _do_get_status(self):
+        """
+        Get status of the device.
+        Input:
+            None
+
+        Output:
+            None
+        """
+        logging.info(__name__ + ' : Get status of the device.')
+        result = self._execute('X')
+        usage = {
+            0: "Channel not in use",
+            1: "Channel used for Nitrogen level",
+            2: "Channel used for Helium Level (Normal pulsed operation)",
+            3: "Channel used for Helium Level (Continuous measurement)",
+            9: "Error on channel (Usually means probe unplugged)"
+        }
+        # current_flowing = {
+        # 0 : "Curent not flowing in Helium Probe Wire",
+        # 1 : "Curent not flowing in Helium Probe Wire"
+        # }
+        # auto_fill_status = {
+        # 00 : "End Fill (Level > FULL)",
+        # 01 : "Not Filling (Level < FULL, Level > FILL)",
+        # 10 : "Filling (Level < FULL, Level > FILL)",
+        # 11 : "Start Filling (Level < FILL)"
+        # }
```

Comment:
@jenshnielsen Not sure why this is here. Probably left for implementation later on.